### PR TITLE
Updating README with cloud IDE info

### PR DESCRIPTION
**Pull request by nate-double-u**
I've been working with [gitpod.io](https://gitpod.io) cloud development environments, and have found it very useful, so I thought I'd make it a more official part of this repo.

* Updating README file to include information about available cloud IDE options for development.

Deploy preview: https://github.com/notaryproject/notaryproject.dev/blob/nw-2021-11-10-cloud-ide-description/README.md

### CODE REVIEW

**Comment by nate-double-u** [comment]
(last force push fixes DCO issue)

File: README.md [hunk 1]
```diff
@@ -36,6 +38,10 @@ $ npm run build
 $ npm run check-links
 ```
 
+### Cloud IDE
+
+This repository has been setup to use a [Gitpod.io](https://gitpod.io) remote development environment. The for example, visiting [https://gitpod.io/#https://github.com/notaryproject/notaryproject.dev](https://gitpod.io/#https://github.com/notaryproject/notaryproject.dev) launches an in browser editor and preview environment for the [NotaryProject.dev](https://notaryproject.dev) site.
+
```
Comment (chalin) [inline README.md:44]: I'd move this to the top (as the first second-level heading), and make it more terse -- maybe something like this:

> ## Cloud build
>
> Visit <https://gitpod.io/#https://github.com/notaryproject/notaryproject.dev> to launch a [Gitpod.io](...) IDE that will allow you to build, preview and make changes to this repo.

Then change the current second-level heading to

> ## Local build

and leave everything else the same.

**Review by chalin — COMMENTED** [review]
See inline comments.

File: README.md (hunk 1 above)
Comment (chalin) [reply README.md:44]: FYI: you'll need to rebase given a small commit I made earlier.

**Comment by nate-double-u** [comment]
Revised the titles as per a better understanding of the feedback. Thanks @chalin!

**Review by SteveLasker — APPROVED** [review]
LGTM

**Pull request merged**